sycl/gdb/libsycl.so-gdb.py

```diff
@@ -1,384 +1,1406 @@
-# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# ===-- sycl/gdb/libsycl.so-gdb.py - SYCL Pretty Printers -----------------===#
+#
+# Part of the LLVM Project, under the Apache license v2.0 with LLVM Exceptions.
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# ===-----------------------------------------------------------------------===#
+#
+# This file has been formatted with the 'black' utility version 23.
+#
+# ===-----------------------------------------------------------------------===#
 
-# Please follow GDB coding standards and use 'black' for formatting:
-# https://sourceware.org/gdb/wiki/Internals%20GDB-Python-Coding-Standards
```

Comment:
A link to the GDB internals coding standard does not apply to LLVM.
The correct link should be:
https://llvm.org/docs/CodingStandards.html#python-version-and-source-code-formatting

I considering updating the link, but all LLVM files need to follow the LLVM coding standard - it seems unnecessary to explicitly state it in each source file. If you feel differently let me know and I'll make the change.

Reply:
> I considering updating the link, but all LLVM files need to follow the LLVM coding standard - it seems unnecessary to explicitly state it in each source file. If you feel differently let me know and I'll make the change.

You are absolutely right, given it is in the coding standard it should be followed. The reason I like it being there is because we don't have a lot of Python scripts, so I am not confident people are aware of the formatting rules. On top of that, we don't have a CI check for python formatting, AFAIK, so the clearer we can make it the better. Not blocking though.

Reply:
> The reason I like it being there is because we don't have a lot of Python scripts

I updated the PR with the requested changes.